app/components/ui/atoms/NavigationDropdown.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import Select, { components } from 'react-select'
+
+import Icon from '../atoms/Icon'
+
+const DropdownArrow = props => (
+  <Icon iconName="ChevronDown" overrideName="dropdownArrow" {...props} />
+)
+
+class NavigationDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      value: this.props.defaultValue,
+    }
+
+    this.customReactSelectStyles = {
+      menuList: (base, state) => ({
+        ...base,
+        padding: 0,
+      }),
+      menu: (base, { placement }) => ({
+        ...base,
+        // remove default space between control & menu
+        marginBottom: 0,
+        // leave room for the bottom border of the Control component to be visible on focus
+        marginTop: this.props.theme.borderWidth,
+      }),
+      option: (base, { isSelected, isFocused }) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorBackground,
+        color: this.props.theme.colorText,
+        paddingTop: this.props.theme.space[3],
+        paddingBottom: this.props.theme.space[3],
+      }),
+      control: (base, { isFocused }) => ({
+        ...base,
+        borderRadius: 0,
+        border: 'none',
+        minHeight: this.props.theme.space[6],
+      }),
+    }
+  }
+
+  render() {
+    const { options } = this.props
+
+    const CustomDropdownIndicator = props => (
+      <components.DropdownIndicator {...props}>
+        <DropdownArrow />
+      </components.DropdownIndicator>
+    )
+
+    return (
+      <Select
+        components={{
+          IndicatorSeparator: null,
+          DropdownIndicator: CustomDropdownIndicator,
+        }}
+        getOptionLabel={({ label }) => label}
+        getOptionValue={({ value }) => value}
+        isSearchable={false}
+        maxMenuHeight={1200} // longer than any phone screen
+        onChange={({ value }) => this.setState({ value })}
+        options={options.filter(({ value }) => value !== this.state.value)}
```

Comment:
I'm not sure we should be filtering the selected value from the designs it looks like the current selected should be top of the select list and `textColor`

Reply:
Good point. I didn't clock that we were supposed to leave the current page in the dropdown.

Asking now, especially based on the lack of colour distinction: https://github.com/elifesciences/elife-xpub/issues/821#issuecomment-434657224

Reply:
Ok, removed the filter based on https://github.com/elifesciences/elife-xpub/issues/821#issuecomment-434665623 :+1: